src/api/protectedApiClient.ts

```diff
@@ -226,9 +226,9 @@ const changePrivilegeLevel = (
 };
 
 const getUserData = (): Promise<UserData> => {
-  return AppAxiosInstance.get(ProtectedApiClientRoutes.GET_USER_DATA)
-    .then((res) => res.data)
-    .catch((err) => err);
+  return AppAxiosInstance.get(ProtectedApiClientRoutes.GET_USER_DATA).then(
+    (res) => res.data,
```

Comment:
📌 Not for this ticket, it seems like we want to remove catching from every one of these route calls so we can add some error condition where they are called.

Reply:
https://code4community-team.monday.com/boards/925293934/views/18904377/pulses/1375952699